1-ECommerce/src/components/MenuOption.tsx

```diff
@@ -0,0 +1,29 @@
+import type { Option } from "../Interfaces"
+
+interface MenuOptionProps {
+    active: boolean;
+    option: Option;
+    onOptionClick: () => void;
+}
```

Comment:
If this `Option` type is only used here, define it here instead of creating a `Interfaces` file